docs/user-guide/configure-certificates.md

```diff
@@ -1,30 +1,77 @@
+### Disable trust store validation of certificates
 
-To enable certificate validation without hostname validation, set `zowe.verifyCertificates: NONSTRICT`. Using this setting, the certificate Common Name or Subject Alternate Name (SAN) is not checked. This facilitates deployment to environments where certificates are valid but do not contain a valid hostname. This configuration is for development purposes only and should not be used for production.
+To disable the trust store validation of southbound certificates (i.e. certificates used for secure communication in the direction from Zowe to the external services or components), set the value `zowe.verifyCertificates: DISABLED` in the `zowe.yaml` configuration file.  A scenario when this is recommended is if the z/OSMF certificate is be self-signed or uses an untrusted CA. In this case, Zowe API Mediation Layer does not recognize the signing authority.  
 
-A proper setup of trust store is mandatory to successfully start Zowe with `zowe.verifyCertificates: STRICT`.
+### Enable certificate validation without hostname validation
 
-## Keystore versus key ring
+To enable certificate validation without hostname validation, set `zowe.verifyCertificates: NONSTRICT`. Using this setting, the certificate Common Name or Subject Alternate Name (SAN) is not checked. Disabling this parameter facilitates deployment to environments where certificates are valid but do not contain a valid hostname. This configuration is for development purposes only and should not be used for production.
 
-Zowe supports certificates that are stored either in a USS directory **Java KeyStore** `.p12` format or else held in a **z/OS Keyring**.  z/OS keystore are the preferred choice for storing certificates where system programmers are already familiar with their operation and usage.  The user ID setting up a keystore and connecting it with certificates requires elevated permissions, and in scenarios where you need to create a Zowe sandbox environment or for testing purposes and your TSO user ID doesn't have authority to manipulate key rings, USS keystores are a good alternative.  
+A proper setup of the trust store is mandatory to successfully start Zowe with `zowe.verifyCertificates: STRICT`.
+
+## Choosing USS Keystore or z/OS Key Ring to store Zowe certificates
+
+Zowe supports certificates that are stored either in a USS directory **Java KeyStore** in the `.p12` format or, alternatively, certificates held in a **z/OS Key ring**. Use of a z/OS keystore is the recommended option for storing certificates if system programmers are already familiar with the certificate operation and usage.
```

Comment:
>Java KeyStore
Are we sure these are Java KeyStores and not just, PKCS12 files? PKCS12 files aren't inherent to Java, we just provide a way to automatically create them using Java. As far as I know, Java KeyStore is a file with a .jks extension, not .p12

Reply:
Correct, the PKCS12 format isn't a Java specific thing.   I know that `keytool` is a Java utility that we sometimes refer to in the docs so it's possible that the **Java** prefeix is a throwback to that.